docs/getting_started.md

```diff
@@ -47,11 +47,15 @@ Here are the steps:
          keyName: app-key
    ```
 
+1. Refer to the [`Datadog Agent examples subfolder`](https://github.com/DataDog/datadog-operator/tree/main/examples/datadogagent) and select an appropriate Datadog Agent configuration file.
+
 1. Deploy the Datadog agent with the above configuration file:
    ```shell
    kubectl apply -f /path/to/your/datadog-agent.yaml
    ```
 
+NOTE: If the Kubernetes cluster that you are deploying the Datadog Agent (as an Operator) to is behind a proxy server, the recommended Datadog Agent configuration file to use will be `datadog-agent-all-with-proxy.yaml`.
```

Comment:
```suggestion
**Note:** If the Kubernetes cluster that you are deploying the Datadog Agent (as an Operator) to is behind a proxy server, the recommended Datadog Agent configuration file to use is `datadog-agent-all-with-proxy.yaml`.
```

Reply:
Hi @urseberry 
Thanks. The issue [is resolved](https://github.com/honghuac/datadog-operator/commit/90a3d1b544d1602dab5bdcf8f34322207ea40c4c).